src/main/java/htsjdk/samtools/BAMStartingAtIteratorFilter.java

```diff
@@ -0,0 +1,43 @@
+package htsjdk.samtools;
```

Comment:
license

Reply:
why not htsjdk.samtools.filter ?

Reply:
To me, this seems to have more affinity with `BAMIteratorFilter' and it's other subclass `BAMQueryMultipleIntervalsIteratorFilter`, which are all in this package. Happy to move it if you still think its better there though.